pyproject.toml

```
@@ -1,5 +1,8 @@
 [build-system]
-requires = ["setuptools>=61.0.0,<69.3.0"]
+requires = [
+    "setuptools>=75.8.1; python_version>'3.8'",
+    "setuptools<75.4.0; python_version<'3.9'",
```

Ditto. I'd also use the same version number for clarity in the environment markers:

```suggestion
    "setuptools>=75.8.1; python_version > '3.8'",
    "setuptools<75.4.0; python_version <= '3.8'",
```